subsys/bluetooth/services/mds.c

```diff
@@ -150,7 +150,7 @@ static ssize_t data_uri_read(struct bt_conn *conn, const struct bt_gatt_attr *at
 {
 	sMemfaultDeviceInfo info;
 	char uri[MDS_MAX_URI_LENGTH];
-	size_t uri_base_length = strlen(MDS_URI_BASE);
+	size_t uri_base_length = strlen(MDS_URI_BASE) + 1;
```

Comment:
This probably adds the string terminator in the middle of the  `uri`. Is it correct?

Reply:
Yes